en-US/Design.xml

```diff
@@ -899,6 +909,40 @@ STEP 14: COMMIT <emphasis>...output omitted...</emphasis> localhost/nexus:latest
 
 			</tgroup>
 
+		</table>
+			 <para>
+				However, if the noun refers to a password, a value, or a status, then instead it flows better to state the object name first, followed by "as the password" as an example.
```

Comment:
```suggestion
				However, in come cases, the sentence might be easier to understand if the noun appears first, or if additional language separates the object name from the noun. 
				For example, if the noun refers to a password, a value, or a status, then consider stating the noun first, including explanatory language between the object name and the noun, or doing both.
```

Reply:
Note that my suggested edit is to create two sentences out of one.

I think the original explanation didn't quite match the examples. For instance, the `redhat` password example -- in this example, the name appears before the noun in both versions, which is the expectation (not an exception). The exception is the additional language between the name and the noun, but that wasn't described. 

And the second example (the variable value) does show using the noun before the name (a value of `tmp`) -- but that's not what the original explanation described. 

I tried to expand the original description to include both types of examples.

I'm noticing that both of these examples are situations where values are being introduced for the first time, which is why the usual format doesn't work -- because "use the X password" or "change to the Y value" implies that this is given information (because of the definitive article). But if this is the first mention, then such phrasing can be confusing for learners, since this information is new to them. My sense is that these exceptions might make the most sense when providing the learner with new information -- but I'm not sure if that's always the case. To stay on the safe side, I didn't try to include that context in my suggested edit. (But thought I would share my thinking anyway! =)